hybris/egl/platforms/wayland/eglplatform_wayland.cpp

```diff
@@ -202,6 +210,36 @@ extern "C" int waylandws_post(EGLNativeWindowType win, void *buffer)
 	return ((WaylandNativeWindow *) eglwin->nativewindow)->postBuffer((ANativeWindowBuffer *) buffer);
 }
 
+/**
+ * Loads libhybris's libEGL at runtime to call hybris_egl_display_get_mapping()
+ */
+static struct _EGLDisplay *_hybris_egl_display_get_mapping(EGLDisplay dpy)
+{
+	static void *libEGL_handle = NULL;
+	static PFNHYBRISEGLDISPLAYGETMAPPINGPROC hybris_egl_display_get_mapping_fn = NULL;
+
+	if (!libEGL_handle) {
+		dlerror();  // cleanup error buffer
+		libEGL_handle = dlopen("libEGL.so.1", RTLD_NOW | RTLD_GLOBAL);
```

Comment:
any reason why this isn't RTLD_LAZY?

Reply:
No specific reason, I just thought it would be safer. Is `RTLD_LAZY` better (faster)? I can try with `RTLD_LAZY` and if you think it's better I will fix.